src/js/contentScript.js

```diff
@@ -145,12 +145,14 @@ if (matchDomain('nzherald.co.nz')) {
 }
 
 if (matchDomain('thestar.com')) {
-    const paywall = document.querySelector('.basic-paywall-new');
-    removeDOMElement(paywall);
-    const tbc = document.querySelectorAll('.text-block-container');
-    for (let i = 0; i < tbc.length; i++) {
-        tbc[i].removeAttribute('style');
-    }
+    setTimeout(function () {
+        const paywall = document.querySelector('.basic-paywall-new');
+        removeDOMElement(paywall);
+        const tbc = document.querySelectorAll('.text-block-container');
+        for (let i = 0; i < tbc.length; i++) {
+            tbc[i].removeAttribute('style');
+        }
+    }, 1000); // Delay (in milliseconds)
```

Comment:
```suggestion
    }, 500); // Delay (in milliseconds)
```

Just checked, the change is great. But I think 500ms is enough.

Reply:
If I disable cache, 5/5 tries fail with 500ms. At 1s, cache disabled, 5/5 tries succeed. I think 1s is safer.